frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -41,6 +41,21 @@
 # pylint: enable=import-error,no-name-in-module
 
 
+LUT_COSTS = {
+    1: 29,
+    2: 33,
+    3: 45,
+    4: 74,
+    5: 101,
+    6: 231,
+    7: 535,
+    8: 1721,
+    9: 3864,
+    10: 8697,
+    11: 19522,
+}
```

Comment:
Yes that's a bit approximate to no consider norm2 and output precision and p-error. We have to find a better solution. But yes for this PR I intend to keep it that way. Maybe I can add a comment explaining where it comes from.
So it's only temporary until we get something better. Also the synthesis tool will use the same constant.
In the long term do you want to call the optimizer to recover theses constants ?

Reply:
I think we can't call the optimizer to resolve these as we'd still miss norm2. The best approach would be to have `FHE(Linalg).bits` operator in the compiler which would directly forwarded to the optimizer so it can decide what's best when selecting parameters. 

But that'd be a very big PR, so it's okay to keep it like this for this PR, with some comments.